ChairManager/ChairManager.cpp

```diff
@@ -350,7 +352,7 @@ void ChairManager::DrawMainWindow(bool* pbIsOpen)
         DrawAssetView();
         DrawLog();
         DrawDLLSettings();
-#ifdef _DEBUG
+#if 1
```

Comment:
Should be disabled for release. Maybe hide behind a cmd line arg?

Reply:
Fixed, but I don't think the menu really has much use outside of development. It's not particularly well maintained lol. Could clean it up in future